src/frontend/src/components/authenticateBox.ts

```diff
@@ -569,22 +523,12 @@ export const authnPages = (i18n: I18n, props: AuthnTemplates) => {
   const templates = authnTemplates(i18n, props);
 
   return {
-    firstTime: (
-      opts: Parameters<typeof templates.firstTime>[0],
-      // TODO: remove this parameter once the landing page is ready.
-      landingPageTemplate: boolean = false
-    ) =>
-      // TODO: Use the landing page template always when ready.
-      page(templates.firstTime(opts), landingPageTemplate),
+    firstTime: (opts: Parameters<typeof templates.firstTime>[0]) =>
+      page(templates.firstTime(opts), true),
     useExisting: (opts: Parameters<typeof templates.useExisting>[0]) =>
-      page(templates.useExisting(opts)),
-    pick: (
-      opts: Parameters<typeof templates.pick>[0],
-      // TODO: remove this parameter once the landing page is ready.
-      landingPageTemplate: boolean = false
-    ) =>
-      // TODO: Use the landing page template always when ready.
-      page(templates.pick(opts), landingPageTemplate),
+      page(templates.useExisting(opts), false),
+    pick: (opts: Parameters<typeof templates.pick>[0]) =>
+      page(templates.pick(opts), true),
```

Comment:
So the parameter is still there and we maintain both versions of the landing page?

Reply:
I'll remove this and clean up the showcase in another PR. Is that ok?

Reply:
Yes, thanks!